sdk/app_cpu1/common/sys/scheduler.c

```diff
@@ -23,18 +23,19 @@ static volatile uint32_t elapsed_usec = 0;
 static bool tasks_running = false;
 static volatile bool scheduler_idle = false;
 
-void scheduler_timer_isr(void *arg)
+void scheduler_tick(void)
```

Comment:
Add a comment above this function which explains that this is called only from the timing manager ISR, so this function executes also runs in the ISR context.